README.md

```diff
@@ -773,6 +987,23 @@ meant to be able to change with it.
     end
     ```
 
+  * <a name="keep-example-descriptions-short"></a>
+    Keep spec description under 40 characters, extract to a context if
```

Comment:
I'm looking at the example docstrings John provided, from the longest, and they fall into several categories:
1. Longest, verbose
```
autocorrects `it "should only fail" do end` to `it "onlies fail" do end`
```
I personally can't figure out why it belongs to the docstring in the first place, as it doesn't add much over the expectation in the example itself, which presumably is:
```ruby
include_examples 'autocorrect',
  'it "should only fail" do end',
  'it "onlies fail" do end'
```

And don't think this rule is applicable to generated docstrings e.g.
```ruby
it %(rewrites "#{old}" as "#{new}") do
```

2. The ones with "when", "with", "without", and "for" that are extractable into a `context`.
Don't think it makes sense to advise to extract to `context` an example that would be the only example in the context just to make example docstrings shorter.

3. "finds `expect_any_instance_of` instead of an instance double" sounds totally legit, and is 60 characters long.

WDYT of relaxing this to 60, and adding notes that it doesn't apply to cases when
1) the example will be the only one in this context (except the case when docstring is significantly longer than 60) and
2) generated docstrings
?

Side observation:
> "onlies fail"
doesn't sound right.

Reply:
BTW I think that
```
expected_rewrites =
  {
    'should return something'            => 'returns something',
# ...
    'should fizz'                        => 'fizzes'
  }
expected_rewrites.each do |old, new|
  it %(rewrites "#{old}" as "#{new}") do
```

goes against [Do not write iterators to generate tests](https://github.com/rubocop-hq/rspec-style-guide/#it-in-iterators), so I would prefer not to mention generated descriptions in this "short example description" rule.

Reply:
Updated the wording and examples, appreciate if you take another look.

Reply:
As per our discussion, updated not to include references to conditionals in docstrings, as there's another rule for that.